web/pages/teams.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Image from 'next/image'
+import PxeLogo from '../public/assets/leagueLogos/pxe_logo.png'
+import {data} from '../data/teams'
+
+function teams() {
+    // let winrate: number[] = []
+    // let rankings: number[] = []
+    // // let winrate: number[] = []
+    // data.map((teamData) => (
+    //     winrate.push(teamData.wins/(teamData.wins + teamData.loses))
+    // ))
+    // winrate.sort(function(a, b){return b - a})\let rankings: number[] = []
+    data.sort(function (a, b) {
```

Comment:
Stylistically, helper functions should be arrow styled
```suggestion
    data.sort((a, b) => {
```